go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -398,9 +405,9 @@ func (be *BuiltinBackupEngine) executeFullBackup(ctx context.Context, params Bac
 		return usable, vterrors.Wrap(err, "can't restart mysqld")
 	}
 
-	// And set read-only mode
-	params.Logger.Infof("resetting mysqld read-only to %v", readOnly)
-	if err := params.Mysqld.SetReadOnly(readOnly); err != nil {
+	// Resetting super_read_only back to its original value
+	params.Logger.Infof("resetting mysqld super_read-only to %v", superReadOnly)
```

Comment:
```suggestion
	params.Logger.Infof("resetting mysqld super-read-only to %v", superReadOnly)
```

Reply:
yeah , I choose '_', I might have missed this place.. so will fix it